### PR TITLE
Extension support start

### PR DESCRIPTION
**Pull request by colw**
Closes #ISSUE

**Description:**

Include extension sign in button under feature flag

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/scripts/boot.js [hunk 1]
```diff
@@ -123,6 +123,14 @@ export const startApp = async (
       store.dispatch(`checkForPersistedSession`)
     })
 
+  window.addEventListener("message", receiveMessage, false)
+  function receiveMessage({ data }) {
+    if (data.type === "LUNIE_EXTENSION") {
+      store.dispatch("setExtensionStatus", true)
+      console.log("Woah! You have the Lunie Extension installed!")
```
Comment (faboweb) [inline src/scripts/boot.js]: ^^

File: src/vuex/modules/extension.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  }
+  const mutations = {
+    setExtensionAvailable(state, enabled) {
+      Vue.set(state, "enabled", enabled)
+    }
+  }
+
+  const actions = {
+    async checkForPresense() {
+      // TODO Check if extension still installed. User could have uninstalled it.
```
Comment (faboweb) [inline src/vuex/modules/extension.js]: True, but that would be very rare case. Do we really need to check this?

File: src/components/common/TmSessionExisting.vue [hunk 3]
```diff
@@ -25,6 +25,14 @@
         title="Use Ledger Nano"
         @click.native="() => setState('hardware')"
       />
+      <LiSession
+        v-if="session.experimentalMode"
+        id="use-extension"
+        icon="laptop"
+        title="Sign in with the Lunie Chrome Extension"
```
Comment (jbibla) [inline src/components/common/TmSessionExisting.vue]:
```suggestion
        title="Use Lunie Chrome extension"
```

**Comment by jbibla** [comment]
this is exciting!!!

File: src/vuex/modules/extension.js (hunk 2 above)
Comment (colw) [reply src/vuex/modules/extension.js]: We should check… somewhere. Doesn't have to be here, or now.

Should we keep the module?

Comment (faboweb) [reply src/vuex/modules/extension.js]: I am open to cleaning up later.

File: src/vuex/modules/session.js [hunk 4]
```diff
@@ -4,6 +4,13 @@ import config from "src/config"
 import { loadKeys, importKey, testPassword } from "../../scripts/keystore.js"
 import { getSeed } from "@lunie/cosmos-keys"
 
+export function extensionListener(store, { data }) {
```
Comment (faboweb) [inline src/vuex/modules/session.js]: should this go into the extension module then?

**Pull request merged**